safe/definitions/minimum_needs.py

```diff
@@ -70,9 +68,9 @@ def _initializes_minimum_needs_fields():
             field_name = '{namespace}__{field_name}'.format(**format_args)
             # It's easier to debug if we have the real value from the
             # postprocessor. That's why I put a Double. ET.
-            field_type = QVariant.Double
+            field_type = qvariant_numbers
```

Comment:
So it brings this kind of discrepancies now : 
![screen shot 2017-02-27 at 15 08 04](https://cloud.githubusercontent.com/assets/1609292/23364280/c2d05d58-fcfe-11e6-8975-fb95c63c6f0b.png)

We have now one transgender displaced.